src/main/java/ch/uzh/ifi/hase/soprafs21/entities/UserInterests.java

```diff
@@ -0,0 +1,67 @@
+package ch.uzh.ifi.hase.soprafs21.entities;
+
+import ch.uzh.ifi.hase.soprafs21.constant.ActivityCategory;
+import ch.uzh.ifi.hase.soprafs21.constant.AgeRange;
+import ch.uzh.ifi.hase.soprafs21.constant.Gender;
+import ch.uzh.ifi.hase.soprafs21.constant.GenderPreference;
+
+import javax.persistence.*;
+import java.io.Serial;
+import java.io.Serializable;
+import java.util.Set;
+
+
+@Entity
+@Table(name = "USER_INTERESTS")
+public class UserInterests implements Serializable {
+
+    @Serial
+    private static final long serialVersionUID = 1L;
+
+    @Id
+    @GeneratedValue
+    private Long id;
+
+    @Column
+    private int ageRangeMin;
+
+    @Column
+    private int ageRangeMax;
+
+    @Enumerated(EnumType.STRING)
+    private GenderPreference genderPreference;
+
+    @ElementCollection(targetClass = ActivityCategory.class)
+    @CollectionTable(name = "USER_INTERESTS_ACITIVITY", joinColumns = @JoinColumn(name = "userInterestsId"))
+    @Column
+    @Enumerated(EnumType.STRING)
+    private Set<ActivityCategory> activityInterests;
```

Comment:
isch das, damit mehreri AcvitityCategories chene inere collection gspeichert werde?

Reply:
Jep

Reply:
okee merci,  @SamuelBruegger vlt chentsch s problem met de List<String> und List<LocalDateTime> be de SchedulingSession eso löse wie das de Amos do gmacht het

Reply:
Yes, using it this way will probably fix the issue. I've created a task #63 and will add it to my pull request.
Btw I have also created a wiki page regarding Hibernate JPA, so we can use it as a quick reference when doing Annotation work in the backend: https://github.com/sopra-fs21-group-16/mth-server/wiki/JPA-Hibernate-Docs